### DISCUSSION TITLE
Topaz questions on difficult data, from an unexperienced user !

### DISCUSSION THREAD

**Discussion by Gia1975** [tbepler/topaz#158] [Q&A]
Hi, 

I am quite new in cryoEM and definitely new in Topaz.
I am trying to use it to pick a multidomain protein 150KDa in Relion4.

I have some questions I would like to ask to Topaz specialists.

1) the particle size in Topaz should be the average size or the max size ?
2) what about the number of expected particles in each micrographs ? for instance, using the Relion logpicker I get an average of 400 particles, should I take this value ?
3) how can I prevent Topaz picking in the carbon ? I carefully removed all the carbon picks from the train model but Topaz keeps picking in the carbon... 

Thanks a lot for your help, 

GIA

**Comment by Gia1975** [comment]
![image](https://user-images.githubusercontent.com/118468801/202514932-686aad24-e3ad-4f39-a291-6e6a36c97817.png)


here is an example on 2 different micrographs, my training set contains ~10000 particles and I have used 500 particles per micrograph, 150A diameter, the autopick FOM here is set to 0

**Comment by tbepler** [comment]
Have you tried adjusting the likelihood threshold for selecting particles (I think FOM in the Relion interface)? Setting this to a larger number will reduce false positives and could be a way to remove the carbon picks. 

When you run extract, I would recommend setting the particle size to that of your largest particle. However, topaz is generally good at localizing the centers, so it tends to work with smaller numbers as well. If you see that topaz is producing multiple picks from a single large particle, then the size should be increased. On the other hand, if you have closely packed particles, increasing the extraction radius size means that you may end up missing some of them, because they are eliminated when their neighbor is selected.

The expected number of particles should be the average number of particles per micrograph, so, yes, 400.

Other than the picks over carbon, are there any clear issues with these picks? It seems like raising the likelihood threshold is probably the easiest and most fruitful thing to try.

**Reply by Gia1975** [reply]
Hi 
Thanks for your reply ! 
the problem is that since I have quite a lot of blue picks in the carbon, by simply raising the FOM level in extraction I am not able to eliminate them automatically. I have to go in the micrographs and remove them one by one... this is surely possible for 100 micrographs but it will be a nightmare if I want to pick my entire dataset (1500 micrographs). However, I noticed that by decreasing the expected number of particles in the training job (let's say to 200), I have almost no picks in the carbon, but I have less particles too, and with lower FOM... Is that a normal behaviour ?
Thanks again !
GIA

**Reply by tbepler** [reply]
Have you checked if these particles over carbon really cause issues for downstream analysis? It could be that they don't impact your reconstruction or that they are easy to filter out with 2d or 3d classification.
